### PR TITLE
CB-3585 migrate itemList to css-modules

### PR DESCRIPTION
**Pull request by teunlao**

### CODE REVIEW

File: webapp/packages/core-blocks/src/ItemList/ItemList.m.css [hunk 1]
```diff
@@ -1,106 +1,93 @@
...
-  }
-  list-item-icon {
+}
+.listItemIcon {
     display: flex;
     align-items: center;
     box-sizing: border-box;
     padding: 8px;
 
     & StaticImage {
```
Comment (Wroud) [inline webapp/packages/core-blocks/src/ItemList/ItemList.m.css]: `.staticImage` ?

File: webapp/packages/core-blocks/src/ItemList/ItemList.m.css [hunk 2]
```diff
@@ -1,106 +1,93 @@
...
+            box-sizing: border-box;
+            overflow: hidden;
+            border-radius: 2px;
+            height: 24px;
+            width: 24px;
+            margin: 0;
 
-        &[name='cross'] {
-          padding: 4px;
+            &[name='cross'] {
```
Comment (Wroud) [inline webapp/packages/core-blocks/src/ItemList/ItemList.m.css]: not sure that it works, does `IconButton` passes `name` property to HTML element?

**Review by Wroud — CHANGES_REQUESTED** [review]

File: webapp/packages/core-blocks/src/ItemList/ItemList.m.css (hunk 1 above)
Comment (teunlao) [reply webapp/packages/core-blocks/src/ItemList/ItemList.m.css]: you're right, fixed it.

File: webapp/packages/core-blocks/src/ItemList/ItemList.m.css (hunk 2 above)
Comment (teunlao) [reply webapp/packages/core-blocks/src/ItemList/ItemList.m.css]: yes, it really doesn't work, changed it to an additional class.
`name` prop is not passed to the HTML

**Review by Wroud — APPROVED** [review]

**Pull request merged**